### PR TITLE
js

### PR DESCRIPTION
**Pull request by Xnrrrrrr**
updates for model, controller and routes

### CODE REVIEW

**Review by AbdullahWins — COMMENTED** [review]
Well Done!

File: backend/controllers/npcController.js
Comment (xEdemo) [inline backend/controllers/npcController.js:1]: Remove file

File: backend/controllers/sentientitemController
Comment (xEdemo) [inline backend/controllers/sentientitemController:1]: Could this be a Boolean on the already existing Item schema?

File: backend/models/Season.js
Comment (xEdemo) [inline backend/models/Season.js:1]: Ref DayNight?

File: backend/routes/bankRoutes.js
Comment (xEdemo) [inline backend/routes/bankRoutes.js:1]: May not need post req

**Review by xEdemo — APPROVED** [review]

**Pull request closed without merging**